### ISSUE TITLE
ERROR in Log file "no app in context: Failed to save avatar for..." 9.0.1

### ISSUE THREAD

**Issue by wawshorty** [owncloud/core#23971]
### Current behaviour
Hello,

after upgrade from OC 8.2.3 to OC 9.0.1 I still get frequently the following error in log file:
"Error 	no app in context 	Failed to save avatar for"
I have had a contact in contacts which had an avatar (this was possible in a previous version of OC). In addition I have a user which has the same name and which have also an avatar. The contact is deleted. The user isn't. I have several users which have an avatar but do not have a problem. So I came to the result that the user or avatar of user is not the problem.
After search I found the following:
`lib/private/avatar.php

class Avatar implements IAvatar {
...
public function getFile($size) {
...
try {
	$file = $this->folder->newFile($path);
	$file->putContent($avatar->data());
} catch (NotPermittedException $e) {
	$this->logger->error('Failed to save avatar for ' . $this->user->getUID());
}`
The try-block was added. That is ok. But what I have to do to avoid the errors in log file?

### Target behaviour
TBD

### Server configuration
**Operating system**:

**Web server:**

**Database:**
MySQL: 5.6.27

**PHP version:**
5.5.33

**ownCloud version:** (see ownCloud admin page)
ownCloud 9.0.1 (stable)

**Updated from an older ownCloud or fresh install:**
manual upgrade from OC 8.2.3 to OC 9.0.1

**Signing status (ownCloud 9.0 and above):**

```

**List of activated apps:**
File, acitvity, galery, clander, contacts

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

**Comment by nickvergessen** [comment]
cc @rullzer @georgehrke

**Comment by wawshorty** [comment]
So, I can give more information. The problem seems to be the user or the avatar of the user. I'm not able to create, update or delete an avatar of a user. For a user which does not haven an avatar yet, it is not possible to add an avatar. Neither via upload nor via selection of already uploaded jpeg-files. For a user which has an avatar it is not possible to delete them nor to update them.

**Comment by rullzer** [comment]
Dup of https://github.com/owncloud/core/issues/22978 I think

**Comment by Wikinaut** [comment]
* OC 9.0.2

I see the same error message in my logfile as https://github.com/owncloud/core/issues/23971#issuecomment-209988014 .

**Comment by rullzer** [comment]
This should be fixed in 9.1 and 9.0.3
Please upgrade once they are released and reopen if not...
